src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -470,6 +471,18 @@ private boolean checkSeparatorInGroup(int groupIdx, boolean isStatic, int line)
     }
 
     /**
+     * Checks whether static imports separated internally.
+     * @param isStatic whether the token is static or not.
+     * @param line the line of the current import.
+     * @return true if static imports are separated internally.
+     */
+    private boolean checkSeparatorInStaticImport(boolean isStatic, int line) {
+        return !beforeFirstImport && (option == ImportOrderOption.TOP
+                || option == ImportOrderOption.BOTTOM) && isStatic
+                && lastImportStatic && line - lastImportLine > 1;
```

Comment:
Simple boolean variables should come before complex conditions like `(option == ImportOrderOption.TOP || option == ImportOrderOption.BOTTOM)`.

Reply:
Done.